client/components/backup-retention-management/test/hooks.tsx

```diff
@@ -1,80 +1,25 @@
-import { renderHook } from '@testing-library/react-hooks';
-import getBackupCurrentSiteSize from 'calypso/state/rewind/selectors/get-backup-current-site-size';
-import getRewindBytesAvailable from 'calypso/state/rewind/selectors/get-rewind-bytes-available';
-import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
-import {
-	useEstimatedCurrentSiteSize,
-	useEstimateSpaceNeeded,
-	usePrepareRetentionOptions,
-} from '../hooks';
+import { getEstimatedCurrentSiteSize, getSpaceNeededInBytes } from '../hooks';
 
 // Constants
-const MOCK_SITE_ID = 123456;
-const MB_IN_BYTES = 2 ** 20;
 const GB_IN_BYTES = 2 ** 30;
 const TB_IN_BYTES = 2 ** 40;
 
-// Mock useSelector
-jest.mock( 'react-redux', () => ( {
-	useSelector: jest.fn( ( func ) => func() ),
-} ) );
-
-// Mock selectors
-jest.mock( 'calypso/state/ui/selectors/get-selected-site-id', () => jest.fn( () => MOCK_SITE_ID ) );
-jest.mock( 'calypso/state/rewind/selectors/get-backup-current-site-size', () => jest.fn() );
-jest.mock( 'calypso/state/rewind/selectors/get-rewind-bytes-available', () => jest.fn() );
-
-// Fixtures
-const RETENTION_OPTIONS_FIXTURES = {
-	CURRENT_PLAN: {
-		label: '30 days',
-		desiredRetentionDays: 30,
-		currentRetentionPlan: 30,
-		checked: false,
-	},
-	DIFFERENT_PLAN: {
-		label: '120 days',
-		desiredRetentionDays: 120,
-		currentRetentionPlan: 30,
-		checked: false,
-	},
-};
-
-// Helper functions
-function renderPrepareRetentionOptionsHook(
-	label: string,
-	desiredRetentionDays: number,
-	retentionPlan: number,
-	checked: boolean
-) {
-	const { result } = renderHook( () =>
-		usePrepareRetentionOptions( label, desiredRetentionDays, retentionPlan, checked )
-	);
-
-	return result.current;
-}
-
 describe( 'hooks', () => {
 	describe( 'useEstimatedCurrentSiteSize', () => {
-		beforeAll( () => {
-			( getSelectedSiteId as jest.Mock ).mockReturnValue( MOCK_SITE_ID );
-		} );
-
 		it.each( [
 			[ 0, 0 ],
 			[ GB_IN_BYTES * 1.25, GB_IN_BYTES ],
 			[ TB_IN_BYTES * 1.25, TB_IN_BYTES ],
 		] )(
 			'should return %d estimated size in bytes, given the last backup size of %d',
 			( expectedEstimation, lastBackupSize ) => {
-				( getBackupCurrentSiteSize as jest.Mock ).mockImplementation( () => lastBackupSize );
-				const { result } = renderHook( () => useEstimatedCurrentSiteSize() );
-				expect( result.current ).toBe( expectedEstimation );
+				const result = getEstimatedCurrentSiteSize( lastBackupSize );
+				expect( result ).toBe( expectedEstimation );
 			}
 		);
 	} );
 
-	describe( 'useEstimateSpaceNeeded', () => {
+	describe( 'getSpaceNeededInBytes', () => {
```

Comment:
These tests are an indication that we don't really need this function and we can just inline the multiplication.